### ISSUE TITLE
"Work place" section not working correctly for admin users

### ISSUE THREAD

**Issue by pettobia** [elan-ev/studip-opencast-plugin#1103]
**Version**
3.21

**Describe the bug**
When logged in as admin user (not root), the opencast section at the "work place" doesn't seem to work properly. 
- some of the preview images aren't shown
- videos appear, which weren't uploaded via Stud.IP (I am not sure, if even root accounts should be able to see them there?)

**Comment by tgloeggl** [comment]
> Thanks for the info. It is not a problem that admins can administrate videos in courses they have access to. But as admin accounts in Stud.IP are always tied to only single institutes, I thought that they would only see the videos that are linked to the institutes the account belongs to.

Yes, that is exactly how this should work, and to give admins a correct list of videos this needs to be reflected on the workplace, i agree.
